dev-requirements.txt

```diff
@@ -24,6 +30,6 @@ sphinx_rtd_theme
 # these two are required by fence -> cirrus -> google_cloud
 pyasn1-modules==0.0.11
 urllib3==1.22
--e git+https://github.com/uc-cdis/cirrus.git@0.0.0#egg=cirrus-0.0.0
+-e git+https://git@github.com/uc-cdis/cirrus.git@0.0.0#egg=cirrus-0.0.0
```

Comment:
Interesting. I haven't seen people type http urls like this for a long time. I think it's probably best to omit the username from the url if it's http. Both of these should install correctly without `git@`

Reply:
:+1:  killed um